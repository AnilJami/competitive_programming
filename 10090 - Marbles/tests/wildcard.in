25
??
???
a
???
a
?b?
e?ee?
eee?c?
e?ee?
?eeeebc
abcc
?a?b?
abbc
?a?b?
a?b?b???c
????a?b?
a???a???
???
a??a?ba
?aaba?
a??a?ba
?aabae?
a??a?ba
?aabae??
?b?e
?b
?b?b
?b
xyz
abc?
ab?bc?
?abbbc
??????b
ababab?
?aabb?ccdd?a?b?cd
aa?bcc?daabbc?d
?aabb?
aa??dd
??????
??
a????
??bb
??ab
aa?a?
a??a
?ab
ab??b?
??aaa
????a?
????bb
