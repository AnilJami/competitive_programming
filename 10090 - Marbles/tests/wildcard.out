match
match
not match
match
not match
not match
match
match
match
match
not match
match
not match
match
not match
match
match
match
not match
match
match
match
not match
not match
not match
